### ISSUE TITLE
Insert parameter should not put quotes around non-string default values

### ISSUE THREAD

**Issue by StephenWeatherford** [microsoft/vscode-azurearmtools#669]
1) Insert a new int parameter with default value of 0.
ACTUAL:
```
        "p4": {
            "type": "int",
            "defaultValue": "0"
        }
```
EXPECTED (no quotes):
```
        "p4": {
            "type": "int",
            "defaultValue": 0
        }
```

2) Insert a new array parameter with default value `[]`
ACTUAL:
```
        "a1": {
            "type": "array",
            "defaultValue": "[]"
        }
```
EXPECTED: no quotes

etc. for object.  Others?

**Comment by nilshedstrom** [comment]
@StephenWeatherford 
I am starting on it now.

**Comment by StephenWeatherford** [comment]
Fixed by #671
